frontend/javascripts/types/schemas/url_state.schema.ts

```diff
@@ -31,7 +31,9 @@ export default {
       enum: ["orthogonal", "oblique", "flight", "volume"],
     },
     "types::MappingType": {
-      enum: ["JSON", "HDF5", null],
+      type: "string",
```

Comment:
```suggestion
      type: ["string", "null"],
```

according to https://stackoverflow.com/questions/16241333/specify-a-value-can-be-a-string-or-null-with-json-schema.
At least that's how I read the `validateUrlStateJSON` method in `validation.ts`.